src/shared/components/Uploader/Uploader.tsx

```diff
@@ -61,8 +71,32 @@ export const Uploader = ({
     e.preventDefault();
   };
 
+  const checkDimensions = (file: File): Promise<boolean> =>
+    new Promise((resolve, reject) => {
+      const reader = new FileReader();
+
+      reader.onload = (e: ProgressEvent<FileReader>) => {
+        const img = new Image();
+        img.onload = () => {
+          const { width, height } = img;
+          resolve(
+            width > MIN_IMAGE_WIDTH &&
```

Comment:
maybe add equal also?
for example width >= MIN_IMAGE_WIDTH && ...

Reply:
By req it is between